components/CropSelector/CropSelector.vue

```diff
@@ -105,6 +105,13 @@ export default {
       type: Boolean,
       default: false,
     },
+    /**
+     * Whether the "+" should appear on the crop dropdown menu.
```

Comment:
I think a better phrasing here would be "Whether the user is allowed to add a new crop or not."